runtime/development/src/lib.rs

```diff
@@ -514,14 +515,14 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 				RuntimeCall::Loans(pallet_loans::Call::close{..}) |
 				// Borrowers should be able to close and execute an epoch
 				// in order to get liquidity from repayments in previous epochs.
-				RuntimeCall::Loans(pallet_loans::Call::update_nav{..}) |
+				RuntimeCall::Loans(pallet_loans::Call::update_portfolio_valuation{..}) |
 				RuntimeCall::PoolSystem(pallet_pool_system::Call::close_epoch{..}) |
 				RuntimeCall::PoolSystem(pallet_pool_system::Call::submit_solution{..}) |
 				RuntimeCall::PoolSystem(pallet_pool_system::Call::execute_epoch{..}) |
 				RuntimeCall::Utility(pallet_utility::Call::batch_all{..}) |
 				RuntimeCall::Utility(pallet_utility::Call::batch{..})
 			),
-			ProxyType::Price => matches!(c, RuntimeCall::Loans(pallet_loans::Call::price { .. })),
+			ProxyType::Price => false,
```

Comment:
Not sure what to put here. Maybe `ProxyType::Price` is no longer needed as a variant?

Reply:
Yeah, now that pricing is built into creation there's no separate step

Reply:
Ok, removed in `development`. 

If I remove it too in `Altair`, do we need a migration for `pallet_proxy` that uses it?